python_src/src/lamp_py/migrations/versions/performance_manager/2e9d81949da8_trips_and_metrics_processing_updates.py

```diff
@@ -0,0 +1,67 @@
+"""trips and metrics processing updates
+
+Revision ID: 2e9d81949da8
+Revises: c514f5b5e389
+Create Date: 2023-04-13 08:41:00.491868
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "2e9d81949da8"
+down_revision = "c514f5b5e389"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column(
+        "vehicle_trips",
+        "trunk_route_id",
+        existing_type=sa.VARCHAR(length=60),
+        nullable=True,
+    )
+    op.alter_column(
+        "vehicle_trips",
+        "stop_count",
+        existing_type=sa.SMALLINT(),
+        nullable=True,
+    )
+    op.add_column(
+        "vehicle_trips",
+        sa.Column("fk_static_timestamp", sa.Integer(), nullable=False),
+    )
+    op.create_foreign_key(
+        "vehicle_trips_fk_static_timestamp_fkey",
+        "vehicle_trips",
+        "static_feed_info",
+        ["fk_static_timestamp"],
+        ["timestamp"],
+    )
+    # ### end Alembic commands ###
```

Comment:
RDS schema migrations:
- make `trunk_route_id` in `vehcile_trips` nullable for new trips loading process
- make `stop_count` in `vehcile_trips` nullable for new trips loading process
- add `fk_static_timestamp` to `vehicle_trips` as foreign key from `timestamp` in `static_feed_info`